sql/core/src/main/scala/org/apache/spark/sql/jdbc/JdbcDialects.scala

```diff
@@ -193,6 +193,26 @@ abstract class JdbcDialect extends Serializable with Logging {
     statement.executeUpdate(s"CREATE TABLE $tableName ($strSchema) $createTableOptions")
   }
 
+  /**
+   * Returns an Insert SQL statement template for inserting a row into the target table via JDBC
+   * conn. Use "?" as placeholder for each value to be inserted.
+   * E.g. `INSERT INTO t ("name", "age", "gender") VALUES (?, ?, ?)`
+   *
+   * @param table The name of the table.
+   * @param fields The fields of the row that will be inserted.
+   * @param isCaseSensitive Whether the table name and field names are case sensitive.
```

Comment:
Can you give some examples of how the implementation should respect this case sensitivity flag? I don't quite get it.

Reply:
@cloud-fan Could we add an option `isCaseSensitive`?

Reply:
we can if it's useful. what's the use case?

Reply:
Maybe we can remove the isCaseSensitive parameter now.
We can add an option `isCaseSensitive` if users find the use case in future.